package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@chargebee/react-native-chargebee",
   "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
remove this line and do the changes on the above line.

Reply:
Done.